test/deck_test.rb

```diff
@@ -11,53 +11,58 @@ def setup
     @card3 = Card.new(:heart, 'Ace', 14)
     @card4 = Card.new(:club, '5', 5)
     @deck = Deck.new(@cards)
+    @deck.add_card(@card1)
+    @deck.add_card(@card2)
+    @deck.add_card(@card3)
```

Comment:
by moving all the `add_card` stuff up here, you can de-duplicate the lines you used later